index.html

```diff
@@ -0,0 +1,258 @@
+                                fill="#6070FF"
+                            ></path>
+                        </svg>
+                    </nav>
+                </nav>
+            </header>
+        </div>
+            <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@500"
+        />
+            <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700" /><link rel="stylesheet" href="https://api.fontshare.com/css?f[]=poppins@400,500,700&display=swap" />
+        <title></title>
+        <link rel="stylesheet" type="text/css" href="styles.css" />
+    </head>
+    <body>
+        <section> <section
+        class="headline-headline"
+    <h1><p
+       class="headline-primary-text"
+>I’m Wachira Nderitu Glad to see you!
+</p></h1>
+<p
+    
+    class="headline-primary-text-1"
+>I’m a software developer! I reside in Nairobi, Kenya. I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+<section
+    
+    class="headline-social-media"
+><section
+    
+    class="headline-supporting-text"
+><p
+    
+    class="headline-letsconnect"
+>LET’S CONNECT</p></div></div>
+<div>
+    <ul class="headline-social-media-icons">
+      <li><a href="#"><img src="twitter-icon.png" alt="Twitter"></a></li>
+      <li><a href="#"><img src="Linkedin-icon.png" alt="LinkedIn"></a></li>
+      <li><a href="#"><img src="mail-icon.png" alt="Email"></a></li>
+      <li><a href="#"><img src="git-icon.png" alt="GitHub"></a></li>
+      <li><a href="#"><img src="chill-icon.png" alt="Chill"></a></li>
+    </ul>
+  </div>
+  <section class="works">
+    <h2 class="cards app-margin">
+        <h2 class="card-container app-bckground-color">
+            <h2 class="card d-flex">
+                <img src="Snapshoot Portfolio.png">
+                <h2 class="works-primary-text"
+                >Tonic
+                </h2>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="counter"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="counter"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>HTML</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="button-see-project" type="button">See Project</button>
```

Comment:
- [ ] [OPTIONAL] I made this review optional because the previous reviewer make it Optional, I would kindly suggest you work on it to make your work card looks great and inline with what we have in the Figma design. If you check closely in the Figma design there is a border around the card that hold all the details and image. 
Thank you.